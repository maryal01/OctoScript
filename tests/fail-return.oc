fn fail-return () -> int {
    return 1;
}

return 1;