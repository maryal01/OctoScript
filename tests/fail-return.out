Fatal error: exception Failure("Can not use return outside of function.")